javascript-arrays/main.js

```diff
@@ -0,0 +1,22 @@
+var colors = ['red', 'white', 'blue'];
+console.log('value of colors[0]:', colors[0]);
+console.log('value of colors[1]:', colors[1]);
+console.log('value of colors[2]:', colors[2]);
+console.log('America is' + ' ' + colors[0] + ',' + ' ' + colors[1] + ',' + ' ' + colors[2] + '.');
+
+colors[2] = 'green';
+console.log('Mexico is' + ' ' + colors[0] + ',' + ' ' + colors[1] + ',' + ' ' + colors[2] + '.');
+
+console.log('value of colors: ', colors);
+
+var students = ['Leo', 'Donnie', 'Raph', 'Mikey'];
+
+var numberOfStudents = students.length;
+console.log('There are' + ' ' + numberOfStudents + ' ' + 'students in the class.');
+
+var lastIndex = numberOfStudents - 1;
+var lastStudent = students[lastIndex];
+
+console.log('The last student in the array is' + ' ' + lastStudent);
+
+console.log(students);
```

Comment:
another pointer: always try to label your console logs, it'll make it easier in the long run!

Reply:
Thanks, got it!)